### PR TITLE
Added support for notifying rabbmiq server

### PR DESCRIPTION
**Pull request by tomaj**
I need notify rabbitmq on one project after deployment so i created recipe. I need one dependency for this - [videlalvaro/php-amqplib](https://github.com/videlalvaro/php-amqplib) than I am not sure if it is ok. If you would like to keep this library without external dependencies, than reject this pull request.

### CODE REVIEW

**Comment by gordalina** [comment]
I think this project must not have dependencies, but you can throw an exception if `PhpAmqpLib\Connection\AMQPConnection` does not exist.

File: recipes/rabbit.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        !isset($config['channel']) ||
+        !isset($config['host']) ||
+        !isset($config['port']) ||
+        !isset($config['username']) ||
+        !isset($config['password']))
+    {
+        throw new \RuntimeException("<comment>Please configure rabbit config:</comment> <info>set('rabbit', array('channel' => 'channel', 'host' => 'host', 'port' => 'port', 'username' => 'username', 'password' => 'password'));</info>");
+    }
+
+    $connection = new \PhpAmqpLib\Connection\AMQPConnection($config['host'], $config['port'], $config['username'], $config['password']);
```
Comment (gordalina) [inline recipes/rabbit.php]: You should `use` FQCN and only instantiate here `AMQPConnection`

File: recipes/rabbit.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    $channel = $connection->channel();
+
+    $msg = new \PhpAmqpLib\Message\AMQPMessage($config['message']);
```
Comment (gordalina) [inline recipes/rabbit.php]: Same hre

**Comment by tomaj** [comment]
I have added use statements and removed depenedency. If you do not install *videlalvaro/php-amqplib* it will fail deployment.

File: recipes/rabbit.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+use PhpAmqpLib\Connection\AMQPConnection;
+use PhpAmqpLib\Message\AMQPMessage;
+
+/**
+ * Notify Rabbit of successful deployment
+ */
+task('deploy:rabbit', function () {
+    
+    if (!class_exists('AMQPConnection')) {
```
Comment (gordalina) [inline recipes/rabbit.php]: Shouldn't you use the FQCN? `PhpAmqpLib\Connection\AMQPConnection `

Comment (tomaj) [inline recipes/rabbit.php]: Yes you are right. I have change it.

**Comment by gordalina** [comment]
Thanks :)

**Comment by gordalina** [comment]
Tagged as 1.0.1

**Pull request merged**